src/utils/makeReactNativeConfig.js

```diff
@@ -226,14 +224,45 @@ function makeReactNativeConfig(
         : userWebpackConfig
     );
 
-    // For simplicity, we don't require users to extend
-    // default config.entry but do it for them.
-    config.entry = defaultWebpackConfig.entry.concat(config.entry);
+    config.entry = injectPolyfillIntoEntry(config.entry, polyfillPath);
```

Comment:
How about merging this inside `Object.assign` above?

Reply:
That would be great